### PR TITLE
chat stream tests

### PR DESCRIPTION
**Pull request by gawi151**
Implements unit tests for `MistralAIClient.chatStream`

### CODE REVIEW

File: test/src/mistralai_client_dart_base_chat_stream_test.dart
Comment (MarcinGruchala) [inline test/src/mistralai_client_dart_base_chat_stream_test.dart:1]: I think we should add a test that checks if the `stream: true` parameter is sent in the request body. 
WDYT?

**Review by MarcinGruchala — APPROVED** [review]
Generally good. The only thing I would add is some test for params that are sent with chat requests. But it can be done after we merge our parts.

File: test/src/mistralai_client_dart_base_chat_stream_test.dart
Comment (gawi151) [reply test/src/mistralai_client_dart_base_chat_stream_test.dart:1]: Oh actually you're right, that's what we should do :) At least the `stream` param.

Comment (gawi151) [reply test/src/mistralai_client_dart_base_chat_stream_test.dart:1]: Added in https://github.com/nomtek/mistralai_client_dart/pull/23/commits/e13932d447727a05e59dce7d0a2c0a8d7fd4a61f

**Review by MarcinGruchala — APPROVED** [review]
Looks good :)

**Pull request merged**